src/main/java/eu/dissco/backend/controller/DigitalMediaObjectController.java

```diff
@@ -76,22 +79,19 @@ public ResponseEntity<JsonApiWrapper> getMultiMediaByIdJsonResponse(
 
   @ResponseStatus(HttpStatus.OK)
   @GetMapping(value = "/{prefix}/{postfix}/annotations", produces = MediaType.APPLICATION_JSON_VALUE)
-  public ResponseEntity<JsonApiListResponseWrapper> getAnnotationsById(
-      @PathVariable("prefix") String prefix, @PathVariable("postfix") String postfix,
-      @RequestParam(defaultValue = DEFAULT_PAGE_NUM) int pageNumber,
-      @RequestParam(defaultValue = DEFAULT_PAGE_SIZE) int pageSize,
-      HttpServletRequest request) {
+  public ResponseEntity<List<AnnotationResponse>> getAnnotationsById(
+      @PathVariable("prefix") String prefix, @PathVariable("postfix") String postfix) {
     var id = prefix + '/' + postfix;
-    String path = SANDBOX_URI + request.getRequestURI();
     log.info("Received get request for annotations on digitalMedia with id: {}", id);
-    var annotations = service.getAnnotationsOnDigitalMediaObject(id, path, pageNumber, pageSize);
+    var annotations = service.getAnnotationsOnDigitalMediaObject(id);
     return ResponseEntity.ok(annotations);
```

Comment:
Shouldn't this be in JsonApi?

Reply:
I reverted this because it changed from the methods in the MediaObjectService and Repo to the annotation. It now makes use of a function which wasn't converted to JSON:API. Changing this will be part of:
https://naturalis.atlassian.net/browse/DD-240